doc/api/fs.md

```diff
@@ -4848,28 +4850,20 @@ added: v10.0.0
 added: v10.0.0
 -->
 
-* `buffer` {Buffer|Uint8Array}
-* `offset` {integer}
-* `length` {integer}
-* `position` {integer}
-* Returns: {Promise}
+* `buffer` {Buffer|Uint8Array} The buffer that the data will be written to.
+* `offset` {integer} The offset in the buffer to start writing at.
+* `length` {integer} The number of bytes to read.
+* `position` {integer} where to begin reading from in the file.
+* Returns: {Promise} Fulfills with an object with a `bytesRead` property
+  specifying the number of bytes read and a `buffer` property that is a
+  reference to the passed in `buffer` argument.
 
 Read data from the file.
 
-`buffer` is the buffer that the data will be written to.
-
-`offset` is the offset in the buffer to start writing at.
-
-`length` is an integer specifying the number of bytes to read.
-
-`position` is an argument specifying where to begin reading from in the file.
 If `position` is `null`, data will be read from the current file position,
 and the file position will be updated.
-If `position` is an integer, the file position will remain unchanged.
 
-Following successful read, the `Promise` is fulfilled with an object with a
-`bytesRead` property specifying the number of bytes read, and a `buffer`
-property that is a reference to the passed in `buffer` argument.
+If `position` is an integer, the file position will remain unchanged.
```

Comment:
If `position = -1`, the file position does change. It's the same behaviour as that of `position = null` because it is substituted with `-1` in `fs.read`. I'm in fact, trying to deprecate `null` here: https://github.com/nodejs/node/pull/37101

Reply:
@RaisinTen care to propose a better phrasing for this bit of the PR? I'm up for changing it

Reply:
@benjamingr done! :)